example/README.md

```diff
@@ -63,4 +63,71 @@ screenshots below).
 
 #### Android
 
-TODO
+You can modify any android native code for RNTP by simply opening the example
+android project in Android Studio and modifying the source:
+
+**macOS Ex**
+
+```sh
+cd react-native-track-player/example
+open -a /Applications/Android\ Studio.app ./android
+```
+
+**NOTE:** remember to run `yarn sync` when you're done making and testing your
+changes to copy them up from `node_modules` to the main repo so they can be
+committed.
+
+###### KotlinAudio
+
+If you need to resolve a bug that exists in `KotlinAudio` you'll need to build
+and install a local version of `KotlinAudio` in order to do so. Here's how:
+
+1. Clone the `KotlinAudio` project:
+
+```sh
+git clone git@github.com:doublesymmetry/KotlinAudio.git
+```
+
+2. Build and export to maven local which is the local dependency repository:
+
+```sh
+cd KotlinAudio
+./gradlew -x test  build publishToMavenLocal
+```
+
+The result of this is a local version of the build published here:
+
+```
+Windows: C:\Users\<user_name>\.m2
+Linux: /home/<user_name>/.m2
+macOS: /Users/<user_name>/.m2
+```
+
+**NOTE**: the `-x test` skips tests for faster build. Make sure you run the test
+before submitting a PR to the `KotlinAudio` project.
+
+3. Point your RNTP dependency at the local build:
+
+```groovy
+// react-native-track-player/android/build.gradle
+...
+
+dependencies {
+//    implementation 'com.github.doublesymmetry:kotlin-audio:0.1.33' // this is remote
+    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.33' // this is local
```

Comment:
```suggestion
//    implementation 'com.github.doublesymmetry:kotlin-audio:0.1.33' // this is local
    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.33' // this is remote
```

@jspizziri The comments seem backward to me. I had to use the hyphenated name to get the local build. 

Also, I had to add `mavenLocal()` to `example/android/build.gradle` (like `allprojects { repositories { mavenLocal() ... } }`). I might be doing something wrong, but I needed those changes (and the fix in #1552) to be able to test with the local build of KotlinAudio.

Reply:
@mattleff thanks for that. I'm going to take a look into that and I'll update the docs as necessary!

Reply:
@mattleff I've just updated the docs based on your feedback. If you have time I'd appreciate if you took another look. Seemed to work for me.